src/v2/view-builder/views/ov/NumberChallengePhoneView.js

```diff
@@ -4,14 +4,14 @@ import hbs from 'handlebars-inline-precompile';
 const NumberChallengePhoneView = View.extend({
   className: 'number-challenge-section',
   template: hbs`
-     <p>
+     <p class="no-translate">
```

Comment:
Why is this marked as "no-translate" given there is an `i18n` message below?

Reply:
Good catch, I am fixing this in my next commit. I just realized that there is a bug in the replacement logic where we are not replacing every instance of the noTranslate content.

The current logic only replaces the first instance of a noTranslationContent.
```
viewText = viewText.replace(noTranslationContent, '');
```

Reply:
![Screen Shot 2021-04-20 at 3 31 42 PM](https://user-images.githubusercontent.com/23267876/115473233-35b1cd80-a1f0-11eb-8be0-9f0f9397ab31.png)

For more context, this is the view where we want to ignore number 10 in 2 places